spec/support/acceptance/cleanup_zabbix.rb

```diff
@@ -0,0 +1,11 @@
+def cleanup_zabbix
+  cleanup_script = <<-SHELL
+  /opt/puppetlabs/bin/puppet resource service zabbix-server ensure=stopped
+  /opt/puppetlabs/bin/puppet resource package zabbix-server-pgsql ensure=purged
+  rm -f /etc/zabbix/.*done
+  if id postgres > /dev/null 2>&1; then
```

Comment:
I labeled this as bug because in the past, the code didn't check if the user exists. The spec tests worked by accident because the first test added the user and the other depended on them. Tests in the past failed if you tried to execute them standalone.